doc/content/reference/configuration/application-server.md

```diff
@@ -52,3 +52,37 @@ Application Server supports communicating the paths of the downlink queue operat
 
 - `as.webhooks.downlink.public-address`: Public address of the HTTP webhooks frontend (default "http://localhost:1885/api/v3")
 - `as.webhooks.downlink.public-tls-address`: Public address of the HTTPS webhooks frontend
+
+## Storage Integration Options
+
+{{< distributions-inline "Cloud" "Enterprise" >}}
```

Comment:
I think it's best to use Enterprise only here, as this configuration is not relevant to Cloud users. Only those operating a Cloud cluster, but that's essentially Enterprise anyway.